frappe/integrations/oauth2_logins.py

```diff
@@ -9,42 +9,42 @@
 
 
 @frappe.whitelist(allow_guest=True)
-def login_via_google(code, state):
+def login_via_google(code: str, state: str):
```

Comment:
can these be `None`? in the case when user declines the oauth prompt

Reply:
I don't know about that. Let me know if these don't work or need more flexibility

Reply:
seems to be fine - don't really see the cancel screen when authorizing


https://user-images.githubusercontent.com/32034600/210314328-cbc93b41-0f7a-4f4c-85cc-4811fdf58fd6.mp4